prboom2/src/heretic/d_main.c

```diff
@@ -24,7 +24,12 @@
 
 static void Heretic_D_DrawTitle(const char *_x)
 {
-  D_SetPage("TITLE", 350, heretic_mus_titl);
+  D_SetPage("TITLE", 210, heretic_mus_titl);
```

Comment:
Why did this change?

Reply:
Because both states showing the title screen have to sum up to 350 tics, but the advisory patch is only displayed after 210 tics:

https://github.com/chocolate-doom/chocolate-doom/blob/0812f49b460239c873fbe46f16717d6aedf04070/src/heretic/d_main.c#L334-L344

Reply:
Ah ok